### PR TITLE
Rework list_builder.py

### PR DESCRIPTION
**Pull request by Salamandar**
FAAAAAASTEEEEEEER

i'm not kidding:
Before: ~3m30
After: ~13s

### CODE REVIEW

**Comment by Salamandar** [comment]
FYI the slowest part of the code is the "git log for first commit that introduced this app".

File: tools/list_builder.py [hunk 1]
```diff
@@ -3,27 +3,34 @@
...
 
+from app_caches import apps_cache_update_all, app_cache_folder  # pylint: disable=import-error
 from packaging_v2.convert_v1_manifest_to_v2_for_catalog import \
     convert_v1_manifest_to_v2_for_catalog  # pylint: disable=import-error
 
+from appslib.utils import (REPO_APPS_ROOT,  # pylint: disable=import-error
+                           get_catalog, git_repo_age)
+
+# Automatically enables error-to-xmpp
+import appslib.xmpplogger  # pylint: disable=import-error
```
Comment (alexAubin) [inline tools/list_builder.py:26]: Kind of nitpicking but ideally we should pick a protocol-agnostic name for this like just "notifications" or idk ... The script is still called `sendxmpppy` but it's historical, under the hood it's actually calling matrix-commander :grimacing:

Comment (Salamandar) [reply tools/list_builder.py:26]: okay :)

Comment (Salamandar) [reply tools/list_builder.py]: Done: logging_sender

**Pull request merged**